### PR TITLE
Author's name removed

### PR DESCRIPTION
**Pull request by samarth2812**

### CODE REVIEW

**Comment by Meldiron** [comment]
Linking issue: https://github.com/appwrite/appwrite/issues/4407

File: tests/LoggerTest.php [hunk 1]
```diff
@@ -6,7 +6,6 @@
  * @subpackage Tests
  *
  * @link https://github.com/utopia-php/framework
- * @author Eldad Fux <eldad@appwrite.io>
```
Comment (Meldiron) [inline tests/LoggerTest.php:9]: Let's remove the whole comment blocks, please.

**Review by Meldiron — CHANGES_REQUESTED** [review]
One tiny comment 🤏 I can mark as approved once resolved.

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by stnguyen90** [comment]
Closing as a stale duplicate

**Pull request closed without merging**